client/docker_creds_.py

```diff
@@ -219,23 +219,27 @@ def _GetUserHomeDir():
     return os.path.expanduser('~')
 
 
-def _GetConfigDirectory():
-  # Return the value of $DOCKER_CONFIG, if it exists, otherwise ~/.docker
-  # see https://github.com/docker/docker/blob/master/cliconfig/config.go
-  if os.environ.get('DOCKER_CONFIG') is not None:
-    return os.environ.get('DOCKER_CONFIG')
-  else:
-    return os.path.join(_GetUserHomeDir(), '.docker')
-
-
 class _DefaultKeychain(Keychain):
   """This implements the default docker credential resolution."""
 
+
+  def GetConfigDirectory(config_dir):
```

Comment:
Does this need to be "def GetConfigDirectory(self, config_dir):" instead? I'm wondering if that is responsible for the travis failure